src/modules/UI/Settings/reducer.js

```diff
@@ -213,7 +322,7 @@ export const settings = (state = initialState, action) => {
       isTouchEnabled: data
     }
   }
-
+*/
```

Comment:
If you are commenting out this action, is it no longer needed?. If so delete it.